src/toolbox.ts

```diff
@@ -799,96 +824,91 @@ export class ZoomPanToolboxItem extends ToolboxItem {
         .ulabel-night #toolbox div.zoom-pan a:active {
             color: white;
         }
-        `
+        `;
         // Create an id so this specific style tag can be referenced
-        const style_id = "zoom-pan-toolbox-item-styles"
+        const style_id = "zoom-pan-toolbox-item-styles";
 
         // Don't add the style tag if its already been added once
-        if (document.getElementById(style_id)) return
+        if (document.getElementById(style_id)) return;
 
         // Grab the document's head and create a style tag
-        const head = document.head || document.querySelector("head")
-        const style = document.createElement('style');
+        const head = document.head || document.querySelector("head");
+        const style = document.createElement("style");
 
         // Add the css and id to the style tag
         style.appendChild(document.createTextNode(css));
-        style.id = style_id
+        style.id = style_id;
 
         // Add the style tag to the document's head
         head.appendChild(style);
     }
 
     private add_event_listeners() {
-        const frames_exist = this.ulabel.config["image_data"].frames.length > 1
+        const frames_exist = this.ulabel.config["image_data"].frames.length > 1;
 
         $(document).on("click.ulabel", ".ulabel-zoom-button", (event) => {
-
             if ($(event.currentTarget).hasClass("ulabel-zoom-out")) {
-                this.ulabel.state.zoom_val /= 1.1
-            }
-            else if ($(event.currentTarget).hasClass("ulabel-zoom-in")) {
-                this.ulabel.state.zoom_val *= 1.1
+                this.ulabel.state.zoom_val /= 1.1;
+            } else if ($(event.currentTarget).hasClass("ulabel-zoom-in")) {
+                this.ulabel.state.zoom_val *= 1.1;
             }
 
-            this.ulabel.rezoom()
+            this.ulabel.rezoom();
 
             // Only try to update the overlay if it exists
-            this.ulabel.filter_distance_overlay?.draw_overlay()
-        })
+            this.ulabel.filter_distance_overlay?.draw_overlay();
+        });
 
         $(document).on("click.ulabel", ".ulabel-pan", (event) => {
             const annbox = $("#" + this.ulabel.config.annbox_id);
             if ($(event.currentTarget).hasClass("ulabel-pan-up")) {
                 annbox.scrollTop(annbox.scrollTop() - 20);
-            }
-            else if ($(event.currentTarget).hasClass("ulabel-pan-down")) {
+            } else if ($(event.currentTarget).hasClass("ulabel-pan-down")) {
```

Comment:
Wrapped vs unwrapped `else` statements. Thoughts?

Reply:
i prefer wrapped, like its doing. ive already switched it to wrapped in several places as ive worked on stuff